eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/LightningMessageTypes.scala

```diff
@@ -121,6 +122,7 @@ case class AcceptChannel(temporaryChannelId: ByteVector32,
                          firstPerCommitmentPoint: PublicKey,
                          tlvStream: TlvStream[AcceptChannelTlv] = TlvStream.empty) extends ChannelMessage with HasTemporaryChannelId {
   val upfrontShutdownScript_opt: Option[ByteVector] = tlvStream.get[ChannelTlv.UpfrontShutdownScript].map(_.script)
+  val channelType_opt: Option[Features] = tlvStream.get[ChannelTlv.ChannelType].map(_.features)
```

Comment:
```suggestion
  val channelType_opt: Option[ChannelType] = tlvStream.get[ChannelTlv.ChannelTypeTlv].map(_.channelType)
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1867/commits/03bf4c363efa65184c39a2af079fb8976f389b77